mock4poc/address_book_mock.yaml

```diff
@@ -0,0 +1,189 @@
+          /demo/addresses/CGNNMO80A02H501R:
+            get:
+              operationId: "getTaxId2"
+              description: "risponde con un indirizzo di piattaforma che fallisce al primo tentativo e un indirizzo generale che funziona sempre"
+              produces:
+                - "application/json"
+              responses:
+                "200":
+                  description: "Ok"
+              x-amazon-apigateway-integration:
+                type: mock
+                httpMethod: GET
+                passthroughBehavior: when_no_match
+                requestTemplates:
+                  application/json: '{"statusCode": 200}'
+                responses:
+                  default:
+                    statusCode: 200
+                    responseTemplates:
+                      application/json: '{ "taxId":"CGNNMO80A02H501R", "digitalAddresses": { "platform":{"type":"PEC","address":"nome2.cognome2@fail-first.demo.it"}, "general":{"type":"PEC","address":"nome1.cognome1@works.demo.it"}}, "courtesyAddresses": [{"type": "EMAIL", "address": "nome2.cognome2@demo2.it"}, {"type": "EMAIL", "address": "nome2.cognome2@demo1.it"}] }'
+          # DEMO default
+          /demo/addresses/{taxId}:
+            get:
+              operationId: "getTaxIdOthers"
+              parameters:
+                - in: path
+                  name: taxId
+                  schema:
+                    type: string
+                  required: true
+              produces:
+                - "application/json"
+              responses:
+                "200":
+                  description: "Ok"
+              x-amazon-apigateway-integration:
+                type: mock
+                httpMethod: GET
+                passthroughBehavior: when_no_match
+                requestTemplates:
+                  application/json: '{"statusCode": 200}'
+                responses:
+                  default:
+                    statusCode: 200
+                    responseTemplates:
+                      application/json: '{ "digitalAddresses": null }'
+
+  AddressBookApiDeployment:
+    Type: AWS::ApiGateway::Deployment
+    Properties:
+      RestApiId: !Ref AddressBookApi
+  AddressBookApiStage:
+    Type: AWS::ApiGateway::Stage
+    Properties:
+      Description: Mock has only a stage
+      RestApiId: !Ref AddressBookApi
+      DeploymentId: !Ref AddressBookApiDeployment
+      StageName: mock
+  
+Outputs:
+  # - API Gateway access informations
+  AddressBookApiUrlMock4Develop:
+    Value: !Sub 'https://${AddressBookApi}.execute-api.${AWS::Region}.amazonaws.com/mock/develop/addresses/'
+    Description: Base URL for API Requests
+    Export:
+      Name: !Sub 'pn-develop-addressbook-api-gw-url'
+  AddressBookApiUrlMock4Demo:
+    Value: !Sub 'https://${AddressBookApi}.execute-api.${AWS::Region}.amazonaws.com/mock/demo/addresses/'
+    Description: Base URL for API Requests
+    Export:
+      Name: !Sub 'demo-main-addressbook-api-gw-url'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Fn::Sub isn&#39;t needed because there are no variables at Outputs/AddressBookApiUrlMock4Demo/Export/Name/Fn::Sub